webCertifications.py

```diff
@@ -127,3 +129,70 @@ def all(self):
                 dbConnection)
 
             return self.showCertifications(message, tools, certifications)
+
+    @cherrypy.expose
+    def v2(self, debug=None):
+        with self.dbConnect() as dbConnection:
+            tools = self.engine.certifications.getAllTools(dbConnection)
+            users = self.engine.certifications.getAllUserList(dbConnection)
```

Comment:
It might help to use the `getInBuildingUserList` func here instead of getting all users.